### PR TITLE
examples: Implement console progress bar for GetObject

### PR DESCRIPTION
**Pull request by harshavardhana**
Progress bar is of the following form

```
Uploading...    2% [=>                         ]   ...
```

### CODE REVIEW

File: examples/GetObjectProgressBar.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      // Check whether the object exists using statObject(). If the object is not found,
+      // statObject() throws an exception. It means that the object exists when statObject()
+      // execution is successful.
+
+      // Get object stat information.
+      ObjectStat objectStat = minioClient.statObject("testbucket", "resumes/4.original.pdf");
+
+      // Get input stream to have content of 'my-objectname' from 'my-bucketname'
+      InputStream is = new ProgressStream("Downloading .. ", ProgressBarStyle.ASCII,
+          objectStat.length(), minioClient.getObject("my-bucketname", "my-objectname"));
```
Comment (danyaljj) [inline examples/GetObjectProgressBar.java]: Looks like this `getObject()` function does not yet exist in the latest deployed version? 
I'm using this: 
```
        <dependency>
            <groupId>io.minio</groupId>
            <artifactId>minio</artifactId>
            <version>3.0.3</version>
        </dependency>
```

Comment (harshavardhana) [inline examples/GetObjectProgressBar.java]: Of course it does @danyaljj getObject has been implemented has been implemented in first release of minio-java. 

What you are looking for is perhaps `ProgressStream()` - this is just an example you should copy paste the `ProgressStream.java` wherever needed and use it with inputStreams to generate progress bar.

Comment (danyaljj) [inline examples/GetObjectProgressBar.java]: what I mean is, `getObject` with two string parameters.

Comment (danyaljj) [inline examples/GetObjectProgressBar.java]: Oh no it's there. Found it.

**Comment by AkshayAwate** [comment]
Can anyone tell me how to implement in python ? @harshavardhana

**Pull request merged**